### PR TITLE
MAT-6906 updated status handler to display error/warnings dynamically

### PR DESCRIPTION
**Pull request by adongare**
## MADiE PR

Jira Ticket: [MAT-6906](https://jira.cms.gov/browse/MAT-6906)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/statusHandler/StatusHandler.tsx [hunk 1]
```diff
@@ -100,34 +100,38 @@ const StatusHandler = ({
...
-                      : failedImport?.patientId;
-                  return (
-                    <li data-testid="failed-test-cases">
-                      {names} <br />
-                      <span tw="ml-4">Reason : {failedImport.message}</span>
-                    </li>
-                  );
-                })}
-              </ul>
+              {failedImports.length > 0 && (
```
Comment (sb-cecilialiu) [inline src/components/statusHandler/StatusHandler.tsx:103]: What about failedImports length is 0? Will there be success message?

Comment (adongare) [reply src/components/statusHandler/StatusHandler.tsx:103]: if nothings failed, I don't think we need status alert. it used to display something like this:

`(10) test case(s) were imported. The following (0) test case(s) could not be imported. Please ensure that your formatting is correct and try again.` that seems lil confusing. we get toast message anyways if import was successful with no errors. this is bit redundant in that case

Comment (adongare) [reply src/components/statusHandler/StatusHandler.tsx:103]: without that condition it looks like this:
image

Comment (sb-cecilialiu) [reply src/components/statusHandler/StatusHandler.tsx:103]: @adongare ok thanks a lot! Looks good!

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/components/statusHandler/StatusHandler.tsx (hunk 1 above)
Comment (adongare) [reply src/components/statusHandler/StatusHandler.tsx:103]: @CeciliaLiu8 We made a few changes to display messages dynamically and also changed the format slightly

**Review by sb-prateekkeerthi — APPROVED** [review]

**Review by sb-prateekkeerthi — APPROVED** [review]

**Pull request merged**